### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by sheylaPozo**
_**In this project, I set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I've learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well, so it is important to know!**_

### CODE REVIEW

File: .github/workflows/.stickler.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  stylelint:
+    # indicate where is the config file for stylelint
+    config: 'stylelint.config.js'
+files:
+  ignore:
+    - 'assets/*'
+# PLEASE DO NOT enable auto fixing options
+# if you need extra support from you linter - do it in your local env as described in README for this config
+
+# find full documentation here: https://stickler-ci.com/docs
```
Comment (Dipeshtwis) [inline .github/workflows/.stickler.yml]: [Optional]

- You don't need to include this file.

File: .github/workflows/.gitignore [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Ignore files related to API keys
+.env
+
+# Ignore Windows system files
+.DS_store
+
+# Ignore SASS config files
+.sass-cache
+
+"This file should be ignored by git"
```
Comment (Dipeshtwis) [inline .github/workflows/.gitignore]: - [x] This file should be in the project root path. and not in any other folder.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @sheylaPozo, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of .gitignore file.
:+1:  Your PR title and description is good.
:+1:  You have used semantic html. :muscle: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/.stickler.yml (hunk 1 above)
Comment (sheylaPozo) [reply .github/workflows/.stickler.yml]: ok

File: .github/workflows/.gitignore (hunk 2 above)
Comment (sheylaPozo) [reply .github/workflows/.gitignore]: ok

**Comment by sheylaPozo** [comment]
I finished adding the required changes, and waiting for approval, thank you !

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations @sheylaPozo :tada: :clinking_glasses: 

**Your project is complete!  This is a convenient point to merge :rocket:**

- - - - - -

**_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._**

Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
_Cheers,_
**[Taofeek Olalere](https://github.com/teekaytech)**
__Happy Coding :smiley:__

**Comment by sheylaPozo** [comment]
thank you for the review :D !!

**Pull request merged**